docs/user-guide/configure.md

```diff
@@ -375,6 +375,35 @@ Patterns are applied against the file path relative to the directory of the conf
 
 Overrides have higher precedence than regular configurations. Multiple overrides within the same config are applied in order. That is, the last override block in a config file always has the highest precedence.
 
+## `processors`
+
+> ⚠️ Warning:
+> This is an experimental feature. The API may change in the future.
```

Comment:
[suggest] Adding a note about the previous `processors` may reduce users' confusion. For example:

```suggestion
> This is an experimental feature. The API may change in the future.
>
> This `processors` property was [removed in 15.0.0](../../migration-guide/to-15.md#removed-processors-configuration-property), but has revived for post-processing. Note that this is different from the previous behavior.
```

Any thoughts?

Reply:
I like it 👍 it helps clarify things for users from v14